wsmaster/che-core-api-workspace-shared/src/main/java/org/eclipse/che/api/workspace/shared/Constants.java

```diff
@@ -118,6 +118,19 @@
    */
   public static final String PERSIST_VOLUMES_ATTRIBUTE = "persistVolumes";
 
+  /**
+   * This attribute enables or disables merging plugins while provisioning a workspace. When
+   * enabled, the plugin broker will be configured to attempt to merge plugins where possible (when
+   * two or more plugins share the same image and do not have conflicting settings). Should be
+   * set/read from {@link WorkspaceConfig#getAttributes}.
```

Comment:
```suggestion
   * set/read from {@link Devfile#getAttributes}.
```
?

Reply:
P.S. I'm Ok with leaving as is as well just to be consistent in that file... But eventually, all these WorkspaceConfig#getAttributes should be changed to Devfile#getAttributes

Reply:
Done, except for `WORKSPACE_TOOLING_EDITOR_ATTRIBUTE` which seems to be a legacy attribute.

Reply:
I also noticed the `editorFree` attribute is in `org.eclipse.che.api.workspace.server.devfile.Constants` instead of this file, which I think is maybe a mistake?

Reply:
I may be wrong but there was some time when Devfile lived in a separate maven module and workspace was not aware of that. So, I assume that now editorFree can be moved here but it's not the purpose of this PR...